backend/src/models/job_enrichment.py

```
@@ -122,5 +125,47 @@ def validate_confidence_by_field(
             raise ValueError("confidence_by_field must be an object")
         return value
 
+    @validates("description_sections")
+    def validate_description_sections(
+        self,
+        key: str,
```

Ruff (0.15.1)

[warning] Unused method argument: `key`

(ARG002)